scripts/scil_compute_local_tracking.py

```diff
@@ -12,6 +12,7 @@
 to the previous direction.
 Algo 'prob': a direction drawn from the empirical distribution function defined
 from the SF.
+Algo 'ptt': Parallel-Transport Tractography. See [1] for more details.
```

Comment:
Should we add a line that says that a smaller step size than usual is recommended with PTT. e.g. 1/5 of prob or det step size???

Reply:
Good point. Maybe not 1/5 of prob, but something along the lines of 0.1-0.2mm or below.